### PR TITLE
bump up telemetry and associated rewiring

### PR DESCRIPTION
**Pull request by VaclavK**
**note** there will be version change as @tabman83 will be publishing another PR

following screenshot comes from AI entry when testing the template
image

### CODE REVIEW

File: src/content/src/WebAPIService/Startup.cs [hunk 1]
```diff
@@ -12,10 +12,11 @@
 using Microsoft.AspNetCore.Mvc.Authorization;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Options;
 using System.Reflection;
+using Eshopworld.Telemetry.Configuration;
 using Microsoft.AspNetCore.Mvc.Filters;
 using Microsoft.OpenApi.Models;
+using TelemetrySettings = Eshopworld.Telemetry.Configuration.TelemetrySettings;
```
Comment (dguerin) [inline src/content/src/WebAPIService/Startup.cs]: Where is the conflict to require this?

Comment (VaclavK) [reply src/content/src/WebAPIService/Startup.cs]: there is same class in devops so to resolve in explicit manner...

**Comment by VaclavK** [comment]
> 
> 
> Looks good and I'll approve when the PR check passes.

this will pass when telemetry is merged in.. was waiting for your review as well

**Comment by VaclavK** [comment]
@dguerin last build now succeeded (https://dev.azure.com/eshopworld/Github%20build/_build/results?buildId=139004&view=results)

**Pull request merged**